src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -502,6 +660,10 @@ class ContentSidebar extends PureComponent<Props, State> {
                                 accessStatsError={accessStatsError}
                                 fileError={fileError}
                                 versionError={versionError}
+                                tasks={tasks}
```

Comment:
At some point we should consider collapsing sections feature wise if we see too many props per feature. Something like what the activity feed already does for its own component

So something like
```jsx
tasks = {{
    tasks: tasks,
    error: tasksError,
    onCreate: onTaskCreate,
    ...
}}
```

Reply:
Agreed, the list is getting quite long